### ISSUE TITLE
Continuous deployment of tagged versions

### ISSUE THREAD

**Issue by enriquetuya** [GlobalFishingWatch/treniformis#14]
We need to work on a continuos deployment of lists once a new version is tagged. Once a tag is done, we want to find a way of having those lists:
* Uploaded to GCS.
* Uploaded to Bigquery

**Comment by geowurster** [comment]
@enriquetuya I would argue that this is not a task for _treniformis_, but rather a task for some other component of the GFW ecosystem.  This repo is not reponsible for putting the data anywhere we need it.  It is responsible for aggregating and small transforms.

**Comment by bitsofbits** [comment]
@enriquetuya, can I close this?

**Comment by enriquetuya** [comment]
@bitsofbits I think the GCS upload is not needed considering the way we have been able to download tags and use them. I don't know about the BQ part, I think we should have that somewhere so we avoid having analysts and partners downloading treniformis and uploading the list themselves.

@davidkroodsma what do you think about this one? Would it be useful if we get a way of uploading the tagged lists to BQ with a good naming convention?
